onediff_diffusers_extensions/examples/text_to_image_sdxl.py

```
@@ -93,7 +93,7 @@
 
 # Compile the pipe
 if args.compiler == "oneflow":
-    base = compile_pipe(base)
+    base.unet = oneflow_compile(base.unet)
```

Ruff (0.8.0)

Undefined name `oneflow_compile`

(F821)